libs/libs-pomfix/lib/fix.js

```diff
@@ -0,0 +1,196 @@
+var es = require('event-stream')
+var combine = require('stream-combiner')
+
+var notNull = function (o) { return o !== null }
+var notUndefined = function (o) { return o !== undefined }
+
+var processPom = function ($) {
+  var descriptor = $('plugin descriptor')
+    .map(function () {
+      var descriptor = $(this).text().trim()
+      var pattern = /mvn:(.*)\/(.*)\/(.*)\/xml\/features/
+      var match = descriptor.match(pattern)
+
+      if (match !== null) {
+        return {
+          group: match[1],
+          artifact: match[2],
+          version: match[3],
+          type: 'xml',
+          classifier: 'features'
+        }
+      }
+    }).get()
+
+  var artifactItem = $('plugin artifactItem')
+    .map(function () {
+      var value = function (tag) {
+        return $(this).find(tag).first().text().trim() || undefined
+      }.bind(this)
```

Comment:
Just use the currentValue argument of the map function, then you won't have to use `this`:
```
.map(function(element){
    var value = function(tag){
          return element.find(tag).first().text().trim() || undefined
    }
```